label_studio/data_manager/models.py

```diff
@@ -22,6 +23,7 @@ class ViewBaseModel(models.Model):
     )
 
     class Meta:
+        ordering = ['order']
```

Comment:
Did we look into how changing the ordering affects the query performance? I think we should make sure that we aren't missing indexes because of the new ORDER BY.

Reply:
Added index for order field.

Reply:
I think a two_column index of `project_id` and `order` would be _best_? I'm not sure the index in just the `order` field does anything? Just a guess, I haven't seen the queryplan for this. It may be an unecessary over-optimization too

Reply:
views is a relatively small table and we return only views for one project per request, so I don't expect performance problems here, I'm not sure we need to dig into query plan. 
I changed index to project + order.